app/components/member/Edit/Memberships.js

```diff
@@ -27,6 +27,7 @@ class MemberEditMemberships extends React.PureComponent {
         ).isRequired
       }).isRequired
     }),
+    isSelf: PropTypes.bool.isRequired,
```

Comment:
I couldn't figure out if this was state or a prop in my cut 😅 

I guess it's derived from props, so only changes when props change, so doesn't need to be tracked separately by the renderer as state? Does that mean isAdmin is also a prop, or something?